src/context/WalletProvider/NativeWallet/components/NativeStart.tsx

```diff
@@ -73,6 +80,29 @@ export const NativeStart = ({ history }: RouteComponentProps) => {
           >
             <Text translation={'walletProvider.shapeShift.start.import'} />
           </Button>
+          {walletMigrationFeatureFlag && (
+            <>
+              <Divider mt={4} />
+              <Flex
+                direction={['column', 'row']}
+                mt={2}
+                pt={4}
+                justifyContent='center'
+                alignItems='center'
+              >
+                <Text translation={'walletProvider.shapeShift.legacy.haveMobileWallet'} />
+                <Button
+                  variant='link'
+                  ml={[0, 1.5]}
+                  borderTopRadius='none'
+                  colorScheme='blue'
+                  onClick={() => history.push('/native/legacy/login')}
+                >
+                  {translate('common.logIn')}
```

Comment:
```suggestion
                  {translate('common.login')}
```

Reply:
I see you changed it in the translation file, but I still see `logIn` here.